client/my-sites/stats/stats-module/expand.jsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+
+export default class StatsModuleExpand extends PureComponent {
```

Comment:
I was going to suggest this would make a good stateless function component, though one downside being it's not possible to have pure stateless function components without using something like `connect` or pulling in [`recompose`](https://github.com/acdlite/recompose/blob/master/docs/API.md#pure).